### DISCUSSION TITLE
how do u do http_load_test?

### DISCUSSION THREAD

**Discussion by sprappcom** [uNetworking/uWebSockets#1758] [Q&A]
i saw this article but not how it is being tested:

https://unetworkingab.medium.com/5-million-http-3-requests-per-second-d8f0074a5367

**Comment by uNetworkingAB** [comment]
It's the same exact run as for TCP or TLS only that both parts are compiled with WITH_QUIC=1 but I'm not 100% this works anymore. Maybe will need to reset history to around when that blog post was written. Or not
